pkgs/top-level/all-packages.nix

```diff
@@ -3752,6 +3752,8 @@ with pkgs;
 
   mmv = callPackage ../tools/misc/mmv { };
 
+  mnist = callPackage ../data/machine-learning/mnist { };
```

Comment:
There is a `### DATA` section further down the file.

Reply:
Done.